### ISSUE TITLE
All fields are null when stardardizing an ASCII text file as 'cobol' format

### ISSUE THREAD

**Issue by yruslan** [AbsaOSS/enceladus#1390]
## Describe the bug
After Standardization is ran on a flat ASCII test file using `rawformat=cobol`, the output dataset contains `null` in all fields, but ones added explicitly by Standardization:

```
scala> spark.read.parquet("/some/path/enceladus_info_date=2020-06-09/enceladus_info_version=1").toJSON.take(1000).foreach(println)
{"errCol":[],"enceladus_record_id":"7477231b-f648-4161-9b8a-34e243bc6921","enceladus_info_date":"2020-06-09","enceladus_info_date_string":"2020-06-09","enceladus_info_version":1}
{"errCol":[],"enceladus_record_id":"e24da98e-50c3-4a26-8fd1-d6580dffc9df","enceladus_info_date":"2020-06-09","enceladus_info_date_string":"2020-06-09","enceladus_info_version":1}
{"errCol":[],"enceladus_record_id":"09b9b30b-7da3-4bc0-a006-801f3d0ea531","enceladus_info_date":"2020-06-09","enceladus_info_date_string":"2020-06-09","enceladus_info_version":1}
{"errCol":[],"enceladus_record_id":"993da28d-566e-4506-a63c-cc8c7ab5b426","enceladus_info_date":"2020-06-09","enceladus_info_date_string":"2020-06-09","enceladus_info_version":1}
{"errCol":[],"enceladus_record_id":"12ae884e-5772-415b-ba91-abc8c71571a5","enceladus_info_date":"2020-06-09","enceladus_info_date_string":"2020-06-09","enceladus_info_version":1}
...
``` 

The same file loads correctly by Cobrix using the same copybook.

## To Reproduce
Run Standardization on a non-empty ASCII text file having 'cobol' as the raw format.

## Expected behaviour
Data should be parsed correctly.

**Comment by yruslan** [comment]
This issue is related to helper scripts only. It is fixed by PR #1389

**Comment by yruslan** [comment]
Release notes
Fixed helper scripts not passing `--cobol-encoding` to Standardization that made it impossible to run with `--cobol-encoding ascii`.
